src/test/java/microsoft/exchange/webservices/data/property/complex/OlsonTimeZoneTest.java

```diff
@@ -37,13 +38,17 @@
   @Test
   public void testOlsonTimeZoneConversion() {
     final String[] timeZoneIds = TimeZone.getAvailableIDs();
-    for (String timeZoneId : timeZoneIds) {
+
+    for (final String timeZoneId : timeZoneIds) {
       if(timeZoneId.startsWith("America") || timeZoneId.startsWith("Europe") || timeZoneId.startsWith("Africa")) {
-        //there are a few timezones that are out of date or don't have direct microsoft mappings according to the Unicode source we use so we will only test Americas, Europe and Africa
-        final OlsonTimeZoneDefinition olsonTimeZone = new OlsonTimeZoneDefinition(TimeZone.getTimeZone(timeZoneId));
-        Assert.assertNotNull(olsonTimeZone.getId());
-      }
+        // There are a few timezones that are out of date or don't have direct microsoft mappings
```

Comment:
Does it make sense to keep the ones that are out of date?  When are these useful?